app/logic/transcript.py

```diff
@@ -11,7 +11,7 @@
 
 def getOtherEventsTranscript(username):
     """
-    Returns a Other Events query object containing all the non-program
+    Returns a Other Events query object containing all the other-program
```

Comment:
while you're at it changing this docstring, it's "Returns **_an_** Other Events query object..."